config/operators/che/subscription.yaml

```diff
@@ -0,0 +1,29 @@
+kind: Namespace
+apiVersion: v1
+metadata:
+  name: toolchain-che
+spec:
+  finalizers:
+    - kubernetes
+---
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  generateName: toolchain-che-
+  namespace: toolchain-che
+spec:
+  targetNamespaces:
+    - toolchain-che
+---
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: eclipse-che
+  namespace: toolchain-che
+spec:
+  channel: stable
+  installPlanApproval: Automatic
+  name: eclipse-che
```

Comment:
I thought that we cannot use the Che operator, but we need to rely on the Che cluster installed manually...

Reply:
What do you mean?
We use Che operator in Hosted Toolchain. Not CRW. And CheCluster is created by the script too: https://github.com/codeready-toolchain/toolchain-infra/pull/12/files#diff-a88c1d3771057755bf3be4f6e244165fR114

Reply:
Just for the clarification. Installing a Che operator is two step process:
- Install the Operator
- Create a CheCluster resource
This is what we did in our member cluster and this is exactly what this script is doing.

Reply:
never mind. I don't know why, but I thought that we didn't use Che operator in the hosted toolchain :man_shrugging: 
It was either at the very beginnning or I got confused with something else :man_facepalming: